src/test/java/io/cdap/plugin/salesforce/etl/SalesforceBatchMultiSourceETLTest.java

```diff
@@ -163,4 +164,25 @@ public void testBlackList() throws Exception {
     Assert.assertEquals(expectedSObjects, actualSObjects);
   }
 
+  @Test
+  public void testValidateSObjects() throws Exception {
```

Comment:
ideally a unit test should be testing only one thing. Can you split this into tests for 
1. allow list(whitelist) with only valid objects
2. allow list with invalid object(s)
3. deny list(blacklist) with only valid sobjects
4. deny list with invalid sobject(s)
Also in one of (2) or (4) you can test multiple invalid sobjects

Reply:
test cases added.

Reply:
With the new changes I see that you can split the unit tests as below.
1. `testValidWhiteListSObjects()` with ```SalesforceMultiSourceConfig salesforceMultiSourceConfig = new SalesforceMultiSourceConfigBuilder().setConsumerKey(CONSUMER_KEY).setConsumerSecret(CONSUMER_SECRET).setUsername(USERNAME).setPassword(PASSWORD).setSecurityToken(SECURITY_TOKEN).setLoginUrl(LOGIN_URL).setWhiteList("Account,Contact").build();```
2. `testValidBlackListSObjects()` with ```SalesforceMultiSourceConfig salesforceMultiSourceConfig = new SalesforceMultiSourceConfigBuilder().setConsumerKey(CONSUMER_KEY).setConsumerSecret(CONSUMER_SECRET).setUsername(USERNAME).setPassword(PASSWORD).setSecurityToken(SECURITY_TOKEN).setLoginUrl(LOGIN_URL).setBlackList("Contact").build();```
3. `testInvalidWhiteListSObjects()` with ```SalesforceMultiSourceConfig salesforceMultiSourceConfig = new SalesforceMultiSourceConfigBuilder().setConsumerKey(CONSUMER_KEY).setConsumerSecret(CONSUMER_SECRET).setUsername(USERNAME).setPassword(PASSWORD).setSecurityToken(SECURITY_TOKEN).setLoginUrl(LOGIN_URL).setWhiteList("invalidObject1,invalidObject2").build();```
4. `testInvalidBlackListSObjects()` with ```SalesforceMultiSourceConfig salesforceMultiSourceConfig = new SalesforceMultiSourceConfigBuilder().setConsumerKey(CONSUMER_KEY).setConsumerSecret(CONSUMER_SECRET).setUsername(USERNAME).setPassword(PASSWORD).setSecurityToken(SECURITY_TOKEN).setLoginUrl(LOGIN_URL).setBlackList("invalidObject1,invalidObject2").build();```

Reply:
added these test cases.